src/EppoClient.php

```diff
@@ -573,6 +573,20 @@ private function checkExpectedType(
             ($expectedVariationType == VariationType::JSON)); // JSON type check un-necessary here.
     }
 
+    /**
+     * @throws EppoClientException
+     */
+    public function fetchAndActivateConfiguration(): void
+    {
+        try {
+            $this->configurationLoader->reloadConfiguration();
+        } catch (HttpRequestException|InvalidApiKeyException|InvalidConfigurationException $e) {
+            if ($this->isGracefulMode) {
+                error_log('[Eppo SDK] Error getting assignment: ' . $e->getMessage());
```

Comment:
I think we want a different error message, as the issue is not with assignments but loading config